docs/cases/cases-manage-settings.asciidoc

```diff
@@ -42,13 +43,14 @@ https://www.elastic.co/subscriptions[appropriate license], and your role needs *
 To create a new connector:
 
 . From the *Incident management system* list, select *Add new connector*.
-. Select the system to send cases to: *{sn}*, *{jira}*, *{ibm-r}*, *{swimlane}*, or *{webhook-cm}*.
+. Select the system to send cases to: *{sn}*, *{jira}*, *{ibm-r}*, *{swimlane}*, *TheHive*, or *{webhook-cm}*.
```

Comment:
Should we make a shared attribute for this, or is it unnecessary in this case?

Reply:
It doesn't hurt!  I created https://github.com/elastic/docs/pull/3076 and https://github.com/elastic/docs-site/pull/45

Reply:
Fixed in fbcf2423c161147e0e3fb3912ad26a59e18805ab